asm/istio/expansion/vm-eastwest-gateway.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: install.istio.io/v1alpha1
```

Comment:
This will be the fourth file in this folder. There's a bash script, two k8s manifests, and now an operator. What's the plan for expansion gateway configuration? Can we consolidate on one method? If all of them are necessary can you put a short README.md in this folder with what each is used for?

Reply:
I can add a README to the files while we consolidate expansion gateway config as a followup.

Reply:
Added a readme to the folder.